I have compared the function methods in https://github.com/OCA/OCB/blob/8.0/addons/stock/stock.py#L679 and https://github.com/OCA/OCB/blob/8.0/addons/stock/stock.py#L1616 and  with correspondant SQL's and seems to be correct.

Very smart method to speed up!

Just a PEP8 remark:

```
./addons/stock/migrations/8.0.1.1/post-migration.py:78:9: E122 continuation line missing indentation or outdented
```

--- a/addons/stock/migrations/8.0.1.1/post-migration.py
+++ b/addons/stock/migrations/8.0.1.1/post-migration.py
@@ -75,7 +75,7 @@ def migrate_move_inventory(cr, registry):
         FROM {stock_inventory_move_rel} rel
         WHERE sm.id = rel.move_id
         """.format(stock_inventory_move_rel=openupgrade.get_legacy_name(
-        'stock_inventory_move_rel')))
+            'stock_inventory_move_rel')))
 
     openupgrade.logged_query(
         cr,

--- a/addons/stock/migrations/8.0.1.1/post-migration.py
+++ b/addons/stock/migrations/8.0.1.1/post-migration.py
@@ -74,9 +74,8 @@ def migrate_move_inventory(cr, registry):
         SET inventory_id = rel.inventory_id
         FROM {stock_inventory_move_rel} rel
         WHERE sm.id = rel.move_id
-        """.format(
-        stock_inventory_move_rel=openupgrade.get_legacy_name(
-            'stock_inventory_move_rel')))
+        """.format(stock_inventory_move_rel=openupgrade.get_legacy_name(
+        'stock_inventory_move_rel')))
 
     openupgrade.logged_query(
         cr,

